infrastructure/modules/serverless/cloudtrail.tf

```diff
@@ -126,14 +109,18 @@ resource "aws_kms_key" "this" {
 EOF
 }
 
+data "aws_kms_alias" "existing" {
```

Comment:
it should be in data.tf

Reply:
+1

Reply:
Added in 621f2fb80404ba20e621b516b17cd84b11fbf1bb